controllers/mustgather/mustgather_controller.go

```diff
@@ -399,6 +385,29 @@ func (r *MustGatherReconciler) getJobFromInstance(instance *mustgatherv1alpha1.M
 	return unstructuredJob, nil
 }
 
+func (r *MustGatherReconciler) getClusterVersionForJobTemplate(name string) (string, error) {
+	clusterVersion := &configv1.ClusterVersion{}
+	err := r.GetClient().Get(context.TODO(), types.NamespacedName{Name: name}, clusterVersion)
+	if err != nil {
+		return "", fmt.Errorf("unable to get clusterversion '%v': %w", name, err)
+	}
+
+	var version string
+	for _, history := range clusterVersion.Status.History {
+		if history.State == "Completed" {
+			version = history.Version
+			break
+		}
+	}
+	if version == "" {
+		return "", goerror.New("unable to determine cluster version from status history")
+	}
+
+	version = version[:strings.LastIndex(version, ".")]
+
+	return version, nil
+}
```

Comment:
Make sure to add RBAC so that the operator is able to get `config.openshift.io/clusterversion` objects https://github.com/openshift/must-gather-operator/blob/master/deploy/02_must-gather-operator.ClusterRole.yaml